src/Service.GraphQLBuilder/Mutations/CreateMutationBuilder.cs

```diff
@@ -126,11 +126,29 @@ private static InputValueDefinitionNode GenerateSimpleInputType(NameNode name, F
                 defaultValue = value.Fields[0].Value;
             }
 
+            ITypeNode type;
+            if (defaultValue is not null)
+            {
+                type = f.Type.NullableType();
```

Comment:
Can ID field be nullable if it has a default value as a built in function in the database?

Reply:
Nope, primary key fields cannot be nullable.